arcade/standard/bed_rush/map.xml

```diff
@@ -0,0 +1,587 @@
+                <all>
+                    <material>bed block</material>
+                </all>
+        </blocks>
+        <team>team-one</team>
+        </all>
+    </deny>
+    <deny id="blue-bed-protect-1">
+        <all>
+            <blocks region="bed-blue-1">
+                <all>
+                    <material>bed block</material>
+                </all>
+            </blocks>
+            <team>team-two</team>
+        </all>
+    </deny>
+    <deny id="blue-bed-protect-2">
+        <all>
+            <blocks region="bed-blue-2">
+                <all>
+                    <material>bed block</material>
+                </all>
+            </blocks>
+            <team>team-two</team>
+        </all>
+    </deny>
+    <all id="red-team-win">
+        <variable var="red_team_win">1</variable>
+        <team>team-one</team>
+    </all>
+    <all id="blue-team-win">
+        <variable var="blue_team_win">1</variable>
+        <team>team-two</team>
+    </all>
+    <all id="red-team-win-final">
+        <variable var="red_team_win">2</variable>
+        <team>team-one</team>
+    </all>
+    <all id="blue-team-win-final">
+        <variable var="blue_team_win">2</variable>
+        <team>team-two</team>
+    </all>
+    <all id="red-team-lose">
+        <variable var="red_team_lose">1</variable>
+        <team>team-one</team>
+    </all>
+    <all id="blue-team-lose">
+        <variable var="blue_team_lose">1</variable>
+        <team>team-two</team>
+    </all>
+    <all id="red-final">
+        <variable var="red_team_win">5</variable>
+        <team>team-one</team>
+    </all>
+    <all id="blue-final">
+        <variable var="blue_team_win">5</variable>
+        <team>team-two</team>
+    </all>
+    <any id="remove-entities">
+        <entity>dropped item</entity>
+        <entity>boat</entity>
+    </any>
+</filters>
+<variables>
+    <variable id="round_number" scope="match" default="1"/>
+    <variable id="round_state" scope="match" default="9"/> <!-- running state, skip cages first round -->
+    <variable id="scoring_team" scope="match" default="0"/>
+    <score id="team_score"/>
+    <with-team id="red_team_score" var="team_score" team="team-one"/>
+    <with-team id="blue_team_score" var="team_score" team="team-two"/>
+    <variable id="player_score" scope="player" default="0"/>
+    <variable id="red_team_win" scope="match" default="0"/>
+    <variable id="red_team_lose" scope="match" default="0"/>
+    <variable id="blue_team_win" scope="match" default="0"/>
+    <variable id="blue_team_lose" scope="match" default="0"/>
+    <variable id="gameover" scope="match" default="0"/>
+</variables>
+<actions>
+    <!-- show to Red team on a round Win -->
+    <trigger filter="red-team-win" action="red-team-win-action" scope="player"/>
+    <action id="red-team-win-action" scope="player">
+        <switch-scope outer="player" inner="match">
+            <switch-scope outer="match" inner="team" filter="only-team-one">
```

Comment:
Why is this a player-scoped action?
The filter is team+match-scoped-variable, meaning a whole team is going to match this, so each player will independently trigger this. 10 players on the team will cause this to trigger 10 times, and because of scope-switching, each of the players will get it 10 times.

```suggestion
    <trigger filter="red-team-win" action="red-team-win-action" scope="team"/>
    <action id="red-team-win-action" scope="team">
```
I'm pretty sure this would already do what you want, since team-one (red) is what would match here

Reply:
Yes, this works.  Thanks.